financial-templates-lib/TwilioTransport.js

```diff
@@ -0,0 +1,15 @@
+const Transport = require("winston-transport");
+const { RoboCaller } = require("./Robo-Caller");
+
+module.exports = class TwilioTransport extends Transport {
+  constructor(opts) {
+    super(opts);
+    this.RoboCaller = new RoboCaller();
+  }
+
+  log(info, callback) {
+    // place the call with the given message.
+    this.RoboCaller.placeCall(`Error reported from ${info.at} with message ${info.message}`);
```

Comment:
Isn't this an async method? If you call it synchronously, won't the async part just be waiting forever until the main thread yields?

Reply:
ye, this is how it is implemented here. the `placeCall` function returns of hangs within ~1 second but you are right it should deal with the promise more gracefully. I refectored this to by async.